packaging/kickstart/kickstart-ostree.ks.template

```diff
@@ -0,0 +1,58 @@
+lang en_US.UTF-8
+keyboard us
+timezone UTC
+text
+reboot
+
+# Partition the disk with hardware-specific boot and swap partitions, adding an
+# LVM volume that contains a 10GB+ system root. The remainder of the volume will
+# be used by the CSI driver for storing data.
+zerombr
+clearpart --all --initlabel
+
+# Create boot and swap partitions as required by the current hardware platform
+reqpart --add-boot
+
+# Add an LVM volume group and allocate a system root logical volume
+part pv.01 --grow
+volgroup rhel pv.01
+logvol / --vgname=rhel --fstype=xfs --size=10240 --name=root
+
+# Lock root user account
+rootpw --lock
+
+# Configure network to use DHCP and activate on boot
+network --bootproto=dhcp --device=link --activate --onboot=on
+
+%pre-install --log=/dev/console --erroronfail
+
+# Create an rpm-ostree authentication file
+mkdir -p /etc/ostree/
+cat > /etc/ostree/auth.json <<'EOF'
+${AUTH_CONFIG}
+EOF
```

Comment:
I like we are doing private registry with auth here! How about also adding a custom CA trust root, in case the private registry has a custom CA cert?
```
# Add custom ca cert to trust root:
mkdir -p /etc/pki/ca-trust/source/anchors/
cat > /etc/pki/ca-trust/source/anchors/2023CertificateAuthorityRHCSv2_Chain.pem << EOF
${AUTH_ADDITIONAL_CA_TRUST}
EOF
update-ca-trust

```

Reply:
If we add this, it should go into a generic section common to all kickstart files, right? 
If we consider this mandatory, I can add this. Otherwise, users can extrapolate on the template and implement this and many other customizations in a similar manner.
WDYT?

Reply:
Its mandatory only if the the registry (bootc) or repo-server (ostree-commit) is using TLS with a custom CA certificate. Which happens quite frequently in enterprise grade use cases / deployments. And it took me a while to figure out where / how to do this.
Its a tricky balance between keeping the docs/example simple and easy, but also realistic and helpfull to use.

Reply:
I'd suggest keeping the current "simplified" content and add appendix in the production docs about any additions we'd like to users to pay attention to.

Reply:
fair! that works for me